homeassistant/components/starlink/sensor.py

```diff
@@ -120,4 +121,18 @@ def native_value(self) -> StateType | datetime:
         native_unit_of_measurement=PERCENTAGE,
         value_fn=lambda data: data.status["pop_ping_drop_rate"] * 100,
     ),
+    StarlinkSensorEntityDescription(
+        key="power",
+        device_class=SensorDeviceClass.POWER,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfPower.WATT,
+        value_fn=lambda data: data.consumption["latest_power"],
```

Comment:
This is coming from history, does that mean it's always going to be one tick behind?

Reply:
No, I believe this is the correct way to get the latest starlink consumption statistics.

Example value of `data.consumption`:
```
{'latest_power': 28.48422622680664, 'mean_power': 31.42145009358724, 'min_power': 22.767833709716797, 'max_power': 40.2529296875, 'total_energy': 0.00785536252339681}
```

Reply:
And I also though that it would be maybe nice to attach rest of the values as attributes but I'm not really sure how to do that elegantly.